### ISSUE TITLE
Faeture Request: Notes section on recipie

### ISSUE THREAD

**Issue by Don-Swanson** [open-eats/OpenEats#96]
It would be nice to have a notes section after the directions (Or be able to add a non-numbered section to the directions)

**Comment by RyanNoelk** [comment]
I see the "Recipe information" as a sort of notes on a recipe. I don't think I'll be adding something like this any time soon. 

Feel free to add an example of what this might look like.

**Comment by Don-Swanson** [comment]
My thought process was more along the lines of private notes to yourself. For example, we have a couple of recipes that are currently in the works in progress state that we have family testing out for us, but the private notes section could be for me to keep about things I have changed or plan to try or something like that.

**Comment by RyanNoelk** [comment]
Ahh ok. So your more looking for a way to keep private notes on a recipe that isn't displayed to others. Basically a way of tracking changes and tests that you have done to the recipe over time.

That makes a lot of since.

**Comment by Don-Swanson** [comment]
Yes, I'm not the best at explaining my ideas, so I'm glad that one made more sense

**Comment by Don-Swanson** [comment]
And Ideally, those notes would be displayed on the recipe to me if I'm logged in, but not to anyone else.

**Comment by djmixman** [comment]
Private notes would be nice, but also a section called "Bakers Notes" or something along those lines would be nice to add to the bottom. I add a "bakers notes" section to my cards to put in things like "Use parchment paper for quicker clean-up" or "A kitchen aide mixer helps with this recipe" or various other notes that doesn't really fit into the description or instructions.
